### DISCUSSION TITLE
Найцікавіші новини місяця: березень 2024

### DISCUSSION THREAD

**Discussion by ikalnytskyi** [shopokodu/community#55] [Теми для подкасту]
* [x]  [xz backdoor](https://gist.github.com/thesamesam/223949d5a074ebc3dce9ee78baad9e27)
* [x] [microsoft хоче безкоштовної пітримки ffmpeg](https://twitter.com/FFmpeg/status/1775178805704888726)
* [ ] [Google: Rust команди вдвічі продуктивніше ніж C++](https://www.reddit.com/r/rust/comments/1bpwmud/media_lars_bergstrom_google_director_of/)
* [ ] [Бйорн Страуструп проти Білого Дому](https://www.infoworld.com/article/3714401/c-plus-plus-creator-rebuts-white-house-warning.html)
* [x] Ліцензійне фіасок Redis. [Microsoft скористались моментом](https://github.com/microsoft/garnet).

**Comment by dolifer** [comment]
https://opensource.diia.gov.ua/ багато де проговорювали, та цікавить ваша думка з цього питання.

**Comment by dolifer** [comment]
https://news.ycombinator.com/item?id=39865810
